### PR TITLE
[1199] Implement Health check

### PR DESCRIPTION
**Pull request by georgschmidtdumont**
Provide a Health Endpoint at /health using Spring Actuator. The custom
GeneratorsHealthCheck sends a request to the /info endpoint of each
registered Generator. If one of the requests is not successfull the
service is demmed down.

Signed-off-by: Schmidt-Dumont Georg (BCI/ESW24) <georg.schmidt-dumont@de.bosch.com>

### CODE REVIEW

**Comment by eclipsewebmaster** [comment]
Issue tracker reference:
https://bugs.eclipse.org/bugs/show_bug.cgi?id=1199

**Comment by eclipsewebmaster** [comment]
Issue tracker reference:
https://bugs.eclipse.org/bugs/show_bug.cgi?id=1199

File: repository/repository-core/src/main/java/org/eclipse/vorto/repository/actuator/GeneratorsHealthCheck.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
+/*
```
Comment (aedelmann) [inline repository/repository-core/src/main/java/org/eclipse/vorto/repository/actuator/GeneratorsHealthCheck.java]: Please add Eclipse header. See other vorto files for reference.

**Comment by aedelmann** [comment]
Hi @georgschmidtdumont 
thanks for the contribution :)
There are some conflicting files . Make sure you rebased with latest state from development. 
Also please check the license header in your source files to be the correct one.

**Pull request merged**